doc/connection_and_messaging_reliability.md

```diff
@@ -18,6 +20,37 @@ In this document you will find information about the reliability aspects of the
 
 - Specific behaviors of the supported transport-protocols (AMQP, MQTT, HTTP).
 
+### Connection Authentication
+
+This is a brief note to clarify about how authentication is done in the IoTHub Device/Module clients.
+
+Authentication of a client in the SDK can be done using either
+- [SAS tokens](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-security#security-tokens), or 
+- [x509 certificates](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-security#supported-x509-certificates), or
+- [Device Provisioning Service](https://docs.microsoft.com/en-us/azure/iot-dps/).
+
+This section does not describe the details of Device Provisioning Service (DPS), please use the link above for details.
+
+When using SAS tokens, authentication can be done by:
+- Providing [your own SAS token](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-security#authentication), or
+- Giving the device keys to the SDK (using the device [connection string](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-device-sdk-c-iothubclient#alternate-device-credentials)) and lettting it create the SAS tokens for you (this is the most usual authentication method).
+
+As mentioned in the articles above, SAS tokens have an expiration time.
```

Comment:
>SAS tokens have an expiration time [](start = 36, length = 34)

Maybe add the default of the SAS token expiry time here.

Reply:
I agree it's more info, but after speaking with Rajeev I see that it's better to have the numbers consolidated somewhere instead of replicating them here.